tests/dataset/test_hbg_dataset.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+import pytest
+import os
+
+import pynwb
+from ipfx.stimulus import StimulusOntology
+from ipfx.dataset.hbg_nwb_data import HBGNWB2Data
+import allensdk.core.json_utilities as ju
+from dictdiffer import diff
+from tests.dataset.test_ephys_nwb_data import nwbfile_to_test
+
+@pytest.fixture
+def tmp_nwb_path(tmpdir_factory):
+    nwb = tmpdir_factory.mktemp("test_nwb_data").join("test_hbg_data.nwb")
+    return str(nwb)
+
+@pytest.fixture
+def hbg_nwb_data(tmp_nwb_path):
+
+    nwbfile = nwbfile_to_test()
+    print(tmp_nwb_path)
+
+    with pynwb.NWBHDF5IO(path=tmp_nwb_path, mode="w") as writer:
+        writer.write(nwbfile)
+
+    ontology =  StimulusOntology([[('name', 'ramp stimulus'), ('code', 'RAMP1')],
```

Comment:
For test the ontology should correspond to the test nwb file.  You use "STIMULUS_CODE" but this code is absence from the ontology.